examples/toggle.rs

```diff
@@ -0,0 +1,42 @@
+//! Uses the StatefulOutputPin embedded_hal trait to toggle the pin
+//! On the stm32 discovery board this is the red led
+//! Target board: STM32F3DISCOVERY
+
+#![deny(unsafe_code)]
+#![no_main]
+#![no_std]
+
+use panic_semihosting as _;
+
+use cortex_m_rt::entry;
+use stm32f3xx_hal::hal::digital::v2::OutputPin;
+use stm32f3xx_hal::prelude::*;
+use stm32f3xx_hal::stm32;
+
+#[entry]
+fn main() -> ! {
+    let p = stm32::Peripherals::take().unwrap();
```

Comment:
To stay consistent with:
https://github.com/stm32-rs/stm32f3xx-hal/blob/902b56f8f11221cb8ffbf24305d66ecea858bf6c/examples/usb_serial.rs#L17

```suggestion
    let dp = stm32::Peripherals::take().unwrap();
```